src/main/Config.cpp

```diff
@@ -1023,659 +1012,609 @@ Config::processConfig(std::shared_ptr<cpptoml::table> t)
-                        "TESTING_MINIMUM_PERSISTENT_ENTRY_LIFETIME < "
-                        "MinimumSorobanNetworkConfig::MINIMUM_PERSISTENT_ENTRY_"
-                        "LIFETIME");
-                }
-
-                LOG_WARNING(
-                    DEFAULT_LOG,
-                    "Overriding MINIMUM_PERSISTENT_ENTRY_LIFETIME to {}",
-                    TESTING_MINIMUM_PERSISTENT_ENTRY_LIFETIME);
-            }
-            else if (item.first == "OVERRIDE_EVICTION_PARAMS_FOR_TESTING")
-            {
-                OVERRIDE_EVICTION_PARAMS_FOR_TESTING = readBool(item);
-            }
-            else if (item.first == "TESTING_EVICTION_SCAN_SIZE")
-            {
-                TESTING_EVICTION_SCAN_SIZE = readInt<uint32_t>(item);
-            }
-            else if (item.first == "TESTING_STARTING_EVICTION_SCAN_LEVEL")
-            {
-                TESTING_STARTING_EVICTION_SCAN_LEVEL =
-                    readInt<uint32_t>(item, 1, BucketList::kNumLevels - 1);
-            }
-            else if (item.first == "TESTING_MAX_ENTRIES_TO_ARCHIVE")
-            {
-                TESTING_MAX_ENTRIES_TO_ARCHIVE = readInt<uint32_t>(item);
-            }
-            else if (item.first == "TESTING_SOROBAN_HIGH_LIMIT_OVERRIDE")
-            {
-                TESTING_SOROBAN_HIGH_LIMIT_OVERRIDE = readBool(item);
-
-                if (TESTING_SOROBAN_HIGH_LIMIT_OVERRIDE)
-                {
-                    LOG_WARNING(DEFAULT_LOG,
-                                "Overriding Soroban limits with "
-                                "TESTING_SOROBAN_HIGH_LIMIT_OVERRIDE");
-                }
-            }
-            else if (item.first == "ARTIFICIALLY_SLEEP_MAIN_THREAD_FOR_TESTING")
-            {
-                ARTIFICIALLY_SLEEP_MAIN_THREAD_FOR_TESTING =
-                    std::chrono::microseconds(readInt<uint32_t>(item));
-            }
-            else if (item.first == "MAX_DEX_TX_OPERATIONS_IN_TX_SET")
-            {
-                auto value = readInt<uint32_t>(item);
-                if (value > 0 && value < MAX_OPS_PER_TX + 2)
-                {
-                    throw std::invalid_argument(fmt::format(
-                        "MAX_DEX_TX_OPERATIONS_IN_TX_SET must be either 0 or "
-                        "at least {} in order to not drop any transactions.",
-                        MAX_OPS_PER_TX + 2));
-                }
-                MAX_DEX_TX_OPERATIONS_IN_TX_SET =
-                    value == 0 ? std::nullopt : std::make_optional(value);
-            }
-            else if (item.first == "EMIT_SOROBAN_TRANSACTION_META_EXT_V1")
-            {
-                EMIT_SOROBAN_TRANSACTION_META_EXT_V1 = readBool(item);
-            }
-            else if (item.first == "EMIT_LEDGER_CLOSE_META_EXT_V1")
-            {
-                EMIT_LEDGER_CLOSE_META_EXT_V1 = readBool(item);
+            std::map<std::string, std::function<void()>> confProcessor = {
```

Comment:
Nit: `unordered_map`?

Reply:
I don't think we care either way here. I went with the shortest version